pipelined/bevy_ui2/src/update.rs

```diff
@@ -53,6 +53,7 @@ fn update_hierarchy(
 #[cfg(test)]
 mod tests {
```

Comment:
I adapted the test to `derive(Component)` independently but it was already done in main, I should replace this file with `crates/bevy_ui/src/update.rs`.

Reply:
Done in faacdd1